libs/ledgerjs/packages/hw-app-btc/src/newops/psbtv2.ts

```diff
@@ -540,8 +540,11 @@ function createKey(buf: Buffer): Key {
   return new Key(buf.readUInt8(0), buf.slice(1));
 }
 function serializeMap(buf: BufferWriter, map: Map<string, Buffer>) {
-  for (const k in map.keys) {
-    const value = map.get(k)!;
+  for (const k of map.keys()) {
+    const value = map.get(k);
+    if (!value) {
+      throw new Error(`Failed to serialize map. Missing value for key ${k}`);
+    }
```

Comment:
Perhaps we can keep the code more compact:

```typescript
  for (const key of map.keys()) {
    const value = map.get(key)!;
```

The `!` only serves the purpose of keeping the Typescript compiler happy, as we know that it's not `Null`.

Reply:
Good point! I'll rebase and make this change in the next day

Reply:
Fixed!